BoxContentSDK/BoxContentSDK/BOXNSURLSessionManager.h

```diff
@@ -76,11 +83,13 @@ NS_ASSUME_NONNULL_BEGIN
 @interface BOXNSURLSessionManager : NSObject
 
 /**
- * A delegate to handle callbacks from session tasks that do not have associated task delegates
- * This is possible if the background tasks were created outside of this BOXNSURLSessionManager (e.g. app restarts)
- * A task delegate can always be re-associated back with a session task by calling associateSessionTaskId:withTaskDelegate:
+ * This method needs to be called once to set up the manager to be ready to perform other public methods
+ * @param delegate  handle callbacks from session tasks that do not have associated task delegates
+ *                  possible if the background tasks were created outside of BOXNSURLSessionManager (e.g. app restarts)
+ *                  A task delegate can always be re-associated with a session task by calling
+ *                  associateSessionTaskId:withTaskDelegate:
  */
-@property (nonatomic, strong, readwrite) id<BOXNSURLSessionManagerDelegate> delegate;
+- (void)setUpWithDelegate:(id<BOXNSURLSessionManagerDelegate>)delegate;
```

Comment:
delegate is a one-time set for nsurlsessions. As long as you use the same BOXNSURLSessionManager, the delegate is the same.